File: linear_search/javascript/linear_search.js
function linearSearch(unOrderedArray,searchElement){
    for(var i=0;i<unOrderedArray.length;i++){
        if(unOrderedArray[i]==searchElement){
            return i;
        };
    }
    return -1;
};

var unOrderedArray=[5,1,290,10,80,50];
var searchElement=10;
console.log(linearSearch(unOrderedArray,searchElement))

File: linear_search/go/linear_search.go
package main

import "fmt"

func linearSearch(numbers []int, item int) int {

	if numbers != nil && len(numbers) > 0 {

		for i := 0; i < len(numbers); i++ {

			if numbers[i] == item {
				return numbers[i]
			}

		}

	}

	return -1
}

func main() {

	numbers := []int{5, 3, 4, 2, 1, 6, 7, 8, 10, 9}

	result := linearSearch(numbers, 7)

	if result != -1 {
		fmt.Println("Item", result, "is found!")
	}
}
